src/Components/LoginSignUp/LoginForm.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useRef } from 'react'
+
+const LoginForm = (props) => {
+    const usernameRef = useRef(null)
+    const passwordRef = useRef(null)
+
+    const onSubmit = () => {
+        props.existingUserOrNot(usernameRef.current.value, passwordRef.current.value)
+    }
+
+    return (
+        <div className='p-5 ms-5 bg-light w-75'>
+            <div className='text-center mb-5'>Login To Your Account</div>
+            <div>
+                <div className="form-group mb-4">
+                    <label >Username</label>
```

Comment:
Again move constants to a separate file.

Reply:
Fixed